src/AddIns/Uno.UI.Lottie.Wasm/ts/Uno.UI.Lottie.ts

```diff
@@ -0,0 +1,255 @@
+				return true;
+			}
+
+			return false;
+		}
+
+		private static updateProperties(
+			runningAnimation: RunningLottieAnimation,
+			newProperties: LottieAnimationProperties) {
+
+			const animation = runningAnimation.animation;
+			const runningProperties = runningAnimation.properties;
+
+			if (runningProperties == null || newProperties.rate != runningProperties.rate) {
+				animation.setSpeed(newProperties.rate);
+			}
+
+			runningAnimation.properties = newProperties;
+		}
+
+		private static createAnimation(properties: LottieAnimationProperties, animationData?: AnimationData): RunningLottieAnimation {
+			const existingAnimation = Lottie._runningAnimations[properties.elementId];
+			if (existingAnimation) {
+				// destroy any previous animation
+				existingAnimation.animation.destroy();
+				existingAnimation.animation = null;
+			}
+
+			const config = Lottie.getPlayerConfig(properties, animationData);
+			const animation = Lottie._player.loadAnimation(config);
+
+			const runningAnimation = {
+				animation: animation,
+				properties: properties
+			};
+
+			Lottie._runningAnimations[properties.elementId] = runningAnimation;
+
+			(animation as any).addEventListener("complete", (e: any) => {
+				Lottie.raiseState(animation);
+			});
+
+			(animation as any).addEventListener("loopComplete", (e: any) => {
+				Lottie.raiseState(animation);
+			});
+
+			(animation as any).addEventListener("segmentStart", (e: any) => {
+				Lottie.raiseState(animation);
+			});
+
+			(animation as any).addEventListener("data_ready", (e: any) => {
+				Lottie._numberOfFrames = animation.totalFrames;
+				Lottie.raiseState(animation);
+			});
+
+			Lottie.raiseState(animation);
+
+			return runningAnimation;
+		}
+
+		private static getStateString(animation: Lottie.AnimationItem): string {
+			const duration = animation.getDuration(false);
+
+			const state = `${animation.animationData.w}|${animation.animationData.h}|` +
+				`${animation.isLoaded}|${animation.isPaused}|${duration}`;
+			return state;
+		}
+
+		private static raiseState(animation: Lottie.AnimationItem) {
+			const element = animation.wrapper;
+			const state = Lottie.getStateString(animation);
+
+			element.dispatchEvent(new CustomEvent("lottie_state", { detail: state }));
+		}
+
+		private static getPlayerConfig(properties: LottieAnimationProperties, animationData?: AnimationData): Lottie.AnimationConfig {
+			let scaleMode = "none";
+			switch (properties.stretch) {
+				case "Uniform":
+					scaleMode = "xMidYMid meet";
+					break;
+				case "UniformToFill":
+					scaleMode = "xMidYMid slice";
+					break;
+				case "Fill":
+					scaleMode = "none";
+					break;
+			}
+
+			const containerElement = (Uno.UI as any).WindowManager.current.getView(properties.elementId);
+
+			// https://github.com/airbnb/lottie-web/wiki/loadAnimation-options
+			const playerConfig: Lottie.AnimationConfig = {
+				loop: true,
+				autoplay: properties.autoplay,
```

Comment:
Codacy found an issue: [The key 'autoplay' is not sorted alphabetically](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)